shell/platform/android/io/flutter/app/FlutterActivity.java

```diff
@@ -151,4 +151,10 @@ public boolean loadIntent(Intent intent) {
     public FlutterView getFlutterView() {
       return flutterView;
     }
+
+    @Override
+    public void onTrimMemory(int level) {
+        if (level == TRIM_MEMORY_RUNNING_LOW)
```

Comment:
Done.
Yes - this level was chosen because it happens while the app is running.  The message sent to the framework is a JSON object in case we need future extensibility to indicate other levels.